ppr-ui/src/components/mhr/index.ts

```diff
@@ -0,0 +1 @@
+
```

Comment:
I think this folder/index might be obsolete now. 
We can add it back in, if we find there is some common components between Mhr Registration and Search perhaps but no purpose at the moment.